ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -148,6 +151,10 @@ open class PillButtonBar: UIScrollView {
         preconditionFailure("init(coder:) has not been implemented")
     }
 
+    @objc public func pillsBackgroundColor(_ backgroundColor: UIColor) {
```

Comment:
This should be removed once you took the approach of the parameter on the initializer (conversation I just resolved)....

Reply:
removed

---
In reply to: [468711518](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r468711518) [](ancestors = 468711518)